### PR TITLE
Store default recording format after "Resore Defaults", fixing lp1857806

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

**Comment by daschuer** [comment]
strange: clang fails with (master merged, rebased later) 
```
In file included from src/sources/soundsourcewv.cpp:4:
src/sources/soundsourcewv.h:9:14: error: typedef redefinition with different types ('void' vs 'struct WavpackContext')
typedef void WavpackContext;
             ^
/usr/local/include/wavpack/wavpack.h:283:31: note: previous definition is here
typedef struct WavpackContext WavpackContext;
```

**Comment by daschuer** [comment]
possible fix is here: https://github.com/mixxxdj/mixxx/pull/2416

**Comment by uklotzde** [comment]
Please add an entry to the change log to closely keep track of bug fixes.

**Comment by daschuer** [comment]
Done.

File: CHANGELOG [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ==== 2.2.4 2020-mm-dd ====
 * Prevent infinite loop when decoding corrupt MP3 files #2417
+* Store default recording format after "Resore Defaults" lp:1857806 #2414
```
Comment (uklotzde) [inline CHANGELOG]: typo

**Review by uklotzde — CHANGES_REQUESTED** [review]

**Comment by daschuer** [comment]
Ups .. I have amended the last commit.

**Comment by daschuer** [comment]
merge?

**Comment by uklotzde** [comment]
Sorry for the delay. We have far too many open PRs.

LGTM. Thank you.

**Pull request merged**